### PR TITLE
first pass through to change up arrow to fix quoting and other minor typos

### LINKED ISSUE #5: Go over all pages and ensure it's up to date
Both text and images. Also check for copy-paste, grammar, and spelling errors

### PR DESCRIPTION
**Pull request by rtanglao**
Partially fixes #5, namely the first 10 files:

- [x] Add-Additional-Users.md , removed miscellaneous z, rest LGTM
- [x] Add-Users.md - created some minor typos, rest LGTM
- [x] Client-Look-and-Feel.md  - fixed some typos and &#8595;
- [x] Frequently-Asked-Questions.md, fixed some typos
- [x] Client-Look-and-Feel.md, fixed some typos
- [x] EMS-Server-With-Custom-Domain.md; fixed some typos
- [x] Get-Your-Own-EMS-Server.md fixed some typos; Down arrow should be &#8595;
- [x] Reset-User-Password.md   fixed some typos;Down arrow should be &#8595;
- [x] Migrate-From-Self-Hosted-to-EMS.md  fixed some typos;Down arrow should be &#8595;
- [x] Set-up-Cross-Signing.md  fixed some typos; Down arrow should be &#8595;

### CODE REVIEW

**Comment by rtanglao** [comment]
- [x] Add-Additional-Users.md , removed miscellaneous z, rest LGTM
- [x] Add-Users.md - created some minor typos, rest LGTM
- [x] Client-Look-and-Feel.md  - fixed some typos and &#8595;
- [x] Frequently-Asked-Questions.html, fixed some typos
- [x] Client-Look-and-Feel., fixed some typos <--- changed down arrow to `&#8595;` which in unicode is `control shift u 2193 enter`
- [x] EMS-Server-With-Custom-Domain; fixed some typos <-- doesn't document homeserver, just  client
- [x] Get-Your-Own-EMS-Server. fixed some typos; Down arrow should be &#8595;
- [x] Reset-User-Password.md   fixed some typos;Down arrow should be &#8595;
- [x] Migrate-From-Self-Hosted-to-EMS.md  fixed some typos;Down arrow should be &#8595;
- [x] Set-up-Cross-Signing.md  fixed some typos; Down arrow should be &#8595;
- [x] Add-Email-to-Your-Account.md <-- July 4, need to change screenshot from staging
- [x] Add-Free-Slack-Bridge.md <-- July 4, change down arrow to `&#8595;<br />`
- [x] Change-Account-Password.md <-- July 4, change down arrow to `&#8595;<br />`
- [x] Check-Status.md <-- July 4, change down arrow to `&#8595;<br />`
- [x] Contribute.md <--- July 4, change `is` to `are`
- [x] Create-a-Conference-Call-in-a-Room.md <-- July 4, change down arrow to `&#8595;<br />`
- [x] Export-and-Import-E2E-Room-Keys.md
- [x] Import-Database-and-Media-Dump.md <-- July 4 no changes
- [x] Leave-Community-On-Dead-Homeserver.md <-- July 4, no changes
- [x] Reset-Cross-Signing.md <-- July 4, change down arrow to `&#8595;<br />`
- [x] Submit-Debug-Logs.md <-- July 4, change down arrow to `&#8595;<br />`
- [x] Summary.md

File: src/Add-Email-to-Your-Account.md [hunk 1]
```diff
@@ -2,26 +2,26 @@
 
 This article is licensed under the standard MIT license. See [Home](index.md) for a full copy.
 
-1. Go to Element `All settings`  
+1. Go to Element `All settings` &#8595;<br />
```
Comment (Twi1ightSparkle) [inline src/Add-Email-to-Your-Account.md:5]: Please undo all of these. Double space is a line break in markdown

**Review by Twi1ightSparkle — CHANGES_REQUESTED** [review]

File: src/Add-Email-to-Your-Account.md (hunk 1 above)
Comment (rtanglao) [reply src/Add-Email-to-Your-Account.md]: done

File: src/Frequently-Asked-Questions.md [hunk 2]
```diff
@@ -28,31 +28,33 @@ For a period in 2019, whilst privacy functionality was being enhanced on vector.
...
+If you have federation enabled for your homeserver, you are also able to bridge into publicly accessible rooms, for example using [https://t2bot.io/](https://t2bot.io/) or integrations available on public homeservers such as matrix.org.
 
 #### How do I add a GitHub integration?
 
-To create a GitHub integration in a room, press the four squares in the top of a room labeled Manage Integrations, accept the privacy policy and select GitHub from the Bots list. Log in to GitHub when prompted and select the repositories and functions you want.
+To create a GitHub integration in a room, click on the `i` icon at the top right, accept the privacy policy, click `Add widgets, bridges & bots`, click `Add integrations` and select GitHub from the Bots list. Log in to GitHub when prompted and select the repositories and functions you want.
 
 Note that your server needs to have federation enabled for integrations to work.
 
 #### How do I add RSS integration to my matrix server?
```
Comment (Twi1ightSparkle) [inline src/Frequently-Asked-Questions.md:55]: Capital Matrix

File: src/Frequently-Asked-Questions.md [hunk 3]
```diff
@@ -128,7 +130,7 @@ You will be able to enter the customization preferences from the managed host pa
 
 * You need to enable the CORS header `Access-Control-Allow-Origin: *` on the web server for the client file. See [https://enable-cors.org/](https://enable-cors.org/) for instructions on how to do this.
 
-The client file need to contain:
+The client file needs to contain:
```
Comment (Twi1ightSparkle) [inline src/Frequently-Asked-Questions.md:133]: Is "needs to" the correct grammar? 🤔 (idk, but it doesn't sound right to me)

File: src/Frequently-Asked-Questions.md [hunk 4]
```diff
@@ -140,7 +142,7 @@ The client file need to contain:
 }
 ```
 
-The server file need to contain:
+The server file needs to contain:
```
Comment (Twi1ightSparkle) [inline src/Frequently-Asked-Questions.md:145]: Same as above

File: src/Frequently-Asked-Questions.md [hunk 5]
```diff
@@ -171,13 +173,13 @@ So, you would need to deprovision an existing server and create a new one, selec
...
 
-Select the "Server Admin" tab and then the "User Info." sub-tab. From here you can search for the relevant user and hit the "Deactivate account button".
+Select the `Server Admin` tab and then the `User Info` sub-tab. From here you can search for the relevant user and hit the `Deactivate account button`.
 
 #### I deleted my host, now my server name is taken and I cannot rebuild
 
 This is actually part of a security measure - We generally prevent hosts from returning to the pool after they were initially claimed in order to prevent people from attempting to imitate old servers / users. Contact support from [https://ems.element.io/support](https://ems.element.io/support) while signed in to get the hostname released.
 
-#### If I in the future want to migrate from EMS to self-hosted, how?
+#### If I in the future I want to migrate from EMS to self-hosted, how?
```
Comment (Twi1ightSparkle) [inline src/Frequently-Asked-Questions.md:182]: "If **I** in the future **I** want to" ?

File: src/Get-Your-Own-EMS-Server.md [hunk 6]
```diff
@@ -4,37 +4,37 @@ This article is licensed under the standard MIT license. See [Home](index.md) fo
...
-1. Enter your name or company name and a password on [https://ems.element.io/user/account](https://ems.element.io/user/account). Then click [Billing address and payment method](https://ems.element.io/user/billing)
+1. Enter your name or company name and a password on [https://ems.element.io/user/account](https://ems.element.io/user/account). Then click [Billing address and payment method](https://ems.element.io/user/billing)   
 ![](images/Screen%20Shot%202020-07-30%20at%202.13.48%20PM.png)
 
-1. Enter your billing address and credit card information on [https://ems.element.io/user/billing](https://ems.element.io/user/billing)  
+1. Enter your billing address and credit card information on [https://ems.element.io/user/billing](https://ems.element.io/user/billing)   
 ![](images/Screen%20Shot%202020-07-30%20at%202.15.49%20PM.png)
 
-1. Click `Your Account`, then `Manage Servers` to get started setting up your EMS server
+1. Click `Your Account`, then `Manage Servers` to get started setting up your EMS server.
```
Comment (Twi1ightSparkle) [inline src/Get-Your-Own-EMS-Server.md:22]: Adding periods at the end of a list item is fine. But if you are doing it, please be consistent and change all pages

File: src/Get-Your-Own-EMS-Server.md [hunk 7]
```diff
@@ -4,37 +4,37 @@ This article is licensed under the standard MIT license. See [Home](index.md) fo
...
-1. Select Monthly or Annual billing and confirm your payment options. Then click Purchase  
+1. Select Monthly or Annual billing and confirm your payment options. Then click Purchase    
 ![](images/Screen%20Shot%202020-07-30%20at%202.21.47%20PM.png)
 
-1. Sit tight for a few minutes while your server is being built  
+1. Sit tight for a few minutes while your server is being built    
 ![](images/Screen%20Shot%202020-07-30%20at%202.23.01%20PM.png)
 
-1. When it's done. Click `Manage hosts` to configure additional settings and add users
+1. When it's done, click `Manage hosts` to configure additional settings and add users.
```
Comment (Twi1ightSparkle) [inline src/Get-Your-Own-EMS-Server.md:39]: Should there be a period at the end when there is a related image right below?

File: src/Submit-Debug-Logs.md [hunk 8]
```diff
@@ -23,36 +23,34 @@ This article is licensed under the standard MIT license. See [Home](index.md) fo
...
-1. Tap the three stacked dots in the top right  
+1. Tap the three stacked dots in the top right   
 ![](images/Screenshot_20200821-144905.png)
 
-1. Tap `Report bug`  
+1. Tap `Report bug`   
 ![](images/Screenshot_20200917-192220.png)
 
-11. Enter a GitHub issue link and a description, make sure `Send logs`, amd `Send crash logs` is checked, then tap the send arrow in the top right  
+11. Enter a GitHub issue link and a description, make sure `Send logs`, amd `Send crash logs` are checked, then tap the send arrow in the top right
```
Comment (Twi1ightSparkle) [inline src/Submit-Debug-Logs.md:55]: amd --> and

**Review by Twi1ightSparkle — CHANGES_REQUESTED** [review]

File: src/Add-Email-to-Your-Account.md (hunk 1 above)
Comment (Twi1ightSparkle) [reply src/Add-Email-to-Your-Account.md]: Click "Resolve conversation" when you have resolved a review comment :D

File: src/Add-Email-to-Your-Account.md [hunk 9]
```diff
@@ -2,26 +2,26 @@
 
 This article is licensed under the standard MIT license. See [Home](index.md) for a full copy.
 
-1. Go to Element `All settings`  
+1. Go to Element `All settings`
```
[the remark this answers is not in the document]
Comment (rtanglao) [reply src/Add-Email-to-Your-Account.md:5]: fixed

File: src/Add-Users.md [hunk 10]
```diff
@@ -2,16 +2,17 @@
...
 
-1. Click 'Add user'. Enter the username and click `Add`  
+1. Click `Add user` Enter the username and click `Add`. 
 ![](images/Screen%20Shot%202020-07-30%20at%202.34.01%20PM.png)
 
-1. Select `Make new user server admin` if you want this user to be able to use the [Synapse Admin API](https://github.com/matrix-org/synapse/tree/master/docs/admin_api) to perform administrative tasks on your server. Be careful with this as this option has a lot of power on your server
+1. Select `Make new user server admin` if you want this user to be able to use the [Synapse Admin API](https://github.com/matrix-org/synapse/tree/master/docs/admin_api) to perform administrative tasks on your server. Be careful with this as this option has a lot of power on your server.
 
-1. The username and password for the new user is displayed. You will only be able to see this information once  
+1. The username and password for the new user is displayed. You will only be able to see this information once so save it in a password manager or some other secure place for future use.
```
[the remark this answers is not in the document]
Comment (rtanglao) [reply src/Add-Users.md:16]: agreed, fixed

File: src/Client-Look-and-Feel.md [hunk 11]
```diff
@@ -2,36 +2,36 @@
...
+  <h2>Rooms to join</h2>
+  <ul>
+      <li><a href='/#/room/#welcome:ems-demo-staging.ems.host'>Welcome to Demo web chat (#welcome)</a></li>
+      <li><a href='/#/room/#support:ems-demo-staging.ems.host'>Support (#support)</a></li>
+      <li><a href='/#/room/#offtopic:ems-demo-staging.ems.host'>Off topic conversation (#offtopic)</a></li>
+  </ul>
+  ```
+  Or you can add more complex HTML and styling.
+
+1. Looks like this in your web browser:
```
[the remark this answers is not in the document]
Comment (rtanglao) [reply src/Client-Look-and-Feel.md:24]: i think my markdown editor corrupted the formatting. fixed

File: src/Create-a-Conference-Call-in-a-Room.md [hunk 12]
```diff
@@ -7,54 +7,54 @@ This article is licensed under the standard MIT license. See [Home](index.md) fo
...
-1. Save the URL if you wish, this is also easily available later. Then click `Save`  
+1. Save the URL if you wish, this is also easily available later. Then click `Save`   
 ![](images/Screen%20Shot%202020-09-15%20at%203.29.46%20PM.png)
 
-1. Click the `X` to close the integration manager  
+1. Click the `X` to close the integration manager   
 ![](images/Screen%20Shot%202020-09-15%20at%203.31.11%20PM.png)
 
-1. To join the room conference, expand the Jitsi section at the top. Then click Join Conference`  
+1. To join the room conference, expand the Jitsi section at the top. Then click Join Conference`
```
[the remark this answers is not in the document]
Comment (rtanglao) [reply src/Create-a-Conference-Call-in-a-Room.md:25]: fixed four spaces globally

File: src/Frequently-Asked-Questions.md (hunk 2 above)
Comment (rtanglao) [reply src/Frequently-Asked-Questions.md:55]: fixed

File: src/Frequently-Asked-Questions.md (hunk 3 above)
Comment (rtanglao) [reply src/Frequently-Asked-Questions.md:133]: reverso.net grammar checker  says "needs to" is correct
image

File: src/Frequently-Asked-Questions.md (hunk 4 above)
Comment (rtanglao) [reply src/Frequently-Asked-Questions.md:145]: same as above :-)

File: src/Frequently-Asked-Questions.md (hunk 5 above)
Comment (rtanglao) [reply src/Frequently-Asked-Questions.md:182]: made the wording less awkward

File: src/Submit-Debug-Logs.md (hunk 8 above)
Comment (rtanglao) [reply src/Submit-Debug-Logs.md:55]: fixed

File: src/Get-Your-Own-EMS-Server.md (hunk 6 above)
Comment (rtanglao) [reply src/Get-Your-Own-EMS-Server.md:22]: took that period out

File: src/Get-Your-Own-EMS-Server.md (hunk 7 above)
Comment (rtanglao) [reply src/Get-Your-Own-EMS-Server.md:39]: i took the period out

File: src/EMS-Server-With-Custom-Domain.md [hunk 13]
```diff
@@ -62,7 +62,7 @@ While not required, you should also add the header `Content-Type application/jso
         expect-ct: max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"
         server: cloudflare
         cf-ray: 5...
-
+
```
[the remark this answers is not in the document]
Comment (rtanglao) [reply src/EMS-Server-With-Custom-Domain.md:65]: fixed, again my markdown editor, typora did this i think?!? anyhow switched to Visual Code and fixed it.

**Review by Twi1ightSparkle — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**